jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/DeflaterTest.java

```diff
@@ -23,11 +23,15 @@
 import java.util.zip.DataFormatException;
 import java.util.zip.Deflater;
 import java.util.zip.Inflater;
-
-import junit.framework.TestCase;
+//import libcore.junit.junit3.TestCaseWithRules;
```

Comment:
Follow the convention for commenting out lines.

Reply:
added J2ObjC comments